src/feature/stats/rephist.c

```diff
@@ -982,6 +984,8 @@ rep_hist_load_mtbf_data(time_t now)
 #define NUM_SECS_BW_SUM_IS_VALID (5*24*60*60)
 /** How many bandwidth usage intervals do we remember? (derived) */
 #define NUM_TOTALS (NUM_SECS_BW_SUM_IS_VALID/NUM_SECS_BW_SUM_INTERVAL)
+/** Start time of bandwidth stats or 0 if we are not collecting the stats*/
+static time_t start_of_bw_stats_interval;
```

Comment:
Please initialise this static variable to 0.

Static variables aren't guaranteed to be 0 on all architectures.